src/Components/PetCard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../Styles/PetCard.scss';
+
+
+const PetCard = () => {
```

Comment:
I very much appreciate you building this out and being explicit about what's going to show on the pet card.

Reply:
This is just generic to get something to show on the page but I appreciate the kudos